client/src/components/Todo-List/TodoList.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { useEffect, useState } from "react";
+import Todo from "../Todo/Todo";
+import "../Todo-List/TodoList.scss";
+
+const TodoList = () => {
+	let [value, setValue] = useState("");
+	let [list, setList] = useState([]);
+	useEffect(() => {
+		getItemsFromServer();
+	}, []);
+
+	const getItemsFromServer = async () => {
+		await getItems().then((todos) => {
+			setList(todos);
+		});
+	};
+
+	const getItems = async () => {
+		const response = await fetch("http://localhost:3000/items");
+		const todos = await response.json();
+		return todos;
+	};
```

Comment:
this shouldn't exist in the react component, think about view and logic separation - why do the view need to know any endpoint or url of a server? or handle the response? (response.json())